plugins/Login/lang/en.json

```diff
@@ -1,68 +1,68 @@
 {
     "Login": {
         "BruteForceLog": "Brute Force Log",
-        "ConfirmationLinkSent": "A confirmation link has been sent to your inbox. Check your e-mail and visit this link to authorize your password change request.",
-        "ContactAdmin": "Possible reason: your host may have disabled the mail() function. <br \/>Please contact your Matomo administrator.",
-        "ExceptionInvalidSuperUserAccessAuthenticationMethod": "A user with Super User access cannot be authenticated using the '%s' mechanism.",
+        "ConfirmationLinkSent": "Open the confirmation link sent to your e-mail inbox to confirm changing your password.",
+        "ContactAdmin": "Possible reason: your host may have disabled the mail() function. <br \/>Please contact your Matomo admin.",
+        "ExceptionInvalidSuperUserAccessAuthenticationMethod": "A user with superuser access cannot be authenticated using the '%s' mechanism.",
         "ExceptionPasswordMD5HashExpected": "The password parameter is expected to be a MD5 hash of the password.",
-        "InvalidNonceToken": "Form security failed, token miss match. Please reload the form and check that your cookies are enabled.",
-        "InvalidNonceReferrer": "Form security failed, invalid referrer header. If you use a proxy server, you must %1$s configure Matomo to accept the proxy header %2$s that forwards the Host header. Also, check that your Referrer header is sent correctly and  If you previously connected using https, please ensure you are connecting over a secure (SSL/TLS) connection and try again.",
-        "InvalidNonceUnexpectedReferrer" : "Form security failed, the referrer header is different from the expected referrer header, check that your Referrer header is sent correctly.",
-        "InvalidNonceOrigin": "Form security failed, invalid origin. If you previously connected using https, please ensure you are connecting over a secure (SSL/TLS) connection and try again.",
-        "InvalidNonceSSLMisconfigured": "Also, you may %1$s force Matomo to use a secure connection%2$s: in your config file %3$s set %4$s below section %5$s",
-        "InvalidOrExpiredToken": "Token is invalid or has expired.",
+        "InvalidNonceToken": "The form security failed because of a token mismatch. Please reload the form and check that your cookies are on.",
+        "InvalidNonceReferrer": "The form security failed because of an invalid referral header. If you are using a proxy server, you must %1$sconfigure Matomo to accept the proxy header%2$s that forwards the host header. Also, check that your referral header is sent correctly. If you previously connected using HTTPS, please ensure you are connecting over a secure (SSL/TLS) connection and try again.",
```

Comment:
This should actually be "referer" https://en.wikipedia.org/wiki/HTTP_referer
but it looks off, and arguably worse in non-translation.
It could be quoted as such though. Something to consider.

Reply:
Removing the whitespaces is fine. There are `a` tags inserted.

I think writing `"Referer"-Header` might be more accurate than using `referral header`